account_invoice_date_due/README.rst

```diff
@@ -0,0 +1,36 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Update Invoice Date Due
+=======================
+
+This module adds a wizard in the invoice menu to allow update the invoice
+date due (Even if it's done).
+
+With this wizard is updated the date due in the invoice and in the journal
```

Comment:
When this wizard is used, the invoice's due date is updated, along with the journal entry.